app/features/app/components/App.js

```diff
@@ -1,12 +1,14 @@
-import React, { Component } from 'react';
+// @flow
+
+import * as React from 'react';
```

Comment:
leave this as it was, please. See what we do in jitsi-meet/react for reference.

Reply:
Types can be imported separately, which we prefer:

```
import React, { Component, type Node } from 'react';
```

> I couldn't find // @flow comment in jitsi-meet/react folder. How are you using flow there?

We use it, a lot: https://github.com/jitsi/jitsi-meet/search?utf8=%E2%9C%93&q=%22%40flow%22&type=

Reply:
> Types can be imported separately, which we prefer:
> import React, { Component, type Node } from 'react';

Cool. I'll use it like this then.

> We use it, a lot:

They are not in all files. When do you use it and when you don't?